mergify_engine/web/asgi.py

```diff
@@ -22,4 +22,5 @@
 if config.SENTRY_URL:
     application = SentryAsgiMiddleware(application)  # type:ignore[assignment]
 
+tracer.configure(service="web")
```

Comment:
I fear this is going to be conflicting soon with the dashboard.
Rather use `engine-web`.

Reply:
This starts to be messy.
For agent and process plugin we use:
  - appname:$HEROKU_APP_NAME
  - service:web/worker

And for logging just :
      service: mergify-engine

Reply:
Well `appname` is Heroku specific and not use across Datadog's product, so we can ignore it basically.

Our current service naming in Datadog is kinda wrong and is not going to give a good experience in APM.

Reply:
gotcha